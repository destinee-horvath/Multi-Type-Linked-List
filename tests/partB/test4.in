NEW 0
INSERT 0 0 tmp_inserted
DELETE 0 0
DELETE 5 0
DELETE 0 1
DELETE 0 100
REMOVE 0
INSERT 0 -1 another
NEW 3
1
2
3
VIEW ALL