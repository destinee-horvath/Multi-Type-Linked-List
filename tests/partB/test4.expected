List 0: 
List 0: tmp_inserted
List 0: 
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
List 0 has been removed.

Number of lists: 0
INVALID COMMAND: INSERT
List 1: 1 -> 2 -> 3
Number of lists: 1
List 1
